arduino/cores/packagemanager/package_manager.go

```diff
@@ -26,6 +26,7 @@ import (
 	"github.com/arduino/arduino-cli/arduino/cores/packageindex"
 	paths "github.com/arduino/go-paths-helper"
 	properties "github.com/arduino/go-properties-orderedmap"
+	"github.com/schollz/closestmatch"
```

Comment:
I would very much prefer if we use `github.com/lithammer/fuzzysearch`, it's used much more and maintained quite well. I know we already use it `closestmatch` somewhere else but nonetheless I'd prefer if we don't keep spreading it in the codebase as am looking to remove it.

Reply:
I left `closestmatch` exactly for the reason that it's already there :slightly_smiling_face: